sections/server.tex

```diff
@@ -43,3 +43,77 @@ \section{QuizFight server}
 requests.
 We then prefer using usernames, which we know by Google's assurance to be
 unique.
+
+\subsection{Client-server communication}
+For communicating with the server we chose the Retrofit library. It is a 
+type-safe REST client for Android (or just Java) developed by Square. 
+It provides a powerful framework for interacting with APIs and sending 
+network requests with \texttt{OkHttp}.
```

Comment:
Maybe it's better to specify what OkHttp is (few words, like: OkHttp is an HTTP client, its request/response API is designed with fluent builders and immutability. It supports both synchronous blocking calls and async calls with callbacks.)